[boschindego] The Bosch Indego Binding doesn't work

The Bosch Indego binding doesn't work since release 2.5.3.
The binding is offline. The user data are correct, with the Bosch Android app it works.
In openHAB I've uninstalled and reinstalled the binding with no effect.
Last year it works without response, but it works (see issue #5975 [https://github.com/openhab/openhab-addons/issues/5975](url)).
The log file says:
2020-04-01 15:46:38.829 [WARN ] [mmon.WrappedScheduledExecutorService] - Scheduled runnable ended with an exception: 
java.lang.NoClassDefFoundError: org/apache/http/impl/client/HttpClients
	at de.zazaz.iot.bosch.indego.IndegoController.connect(IndegoController.java:116) ~[?:?]
	at org.openhab.binding.boschindego.internal.handler.BoschIndegoHandler.poll(BoschIndegoHandler.java:110) ~[?:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_232]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[?:1.8.0_232]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[?:1.8.0_232]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[?:1.8.0_232]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_232]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_232]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_232]
Caused by: java.lang.ClassNotFoundException: org.apache.http.impl.client.HttpClients cannot be found by org.openhab.binding.boschindego_2.5.3
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:433) ~[org.eclipse.osgi-3.12.100.jar:?]
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:395) ~[org.eclipse.osgi-3.12.100.jar:?]
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:387) ~[org.eclipse.osgi-3.12.100.jar:?]
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:150) ~[org.eclipse.osgi-3.12.100.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_232]
	... 9 more

I use a NUC with Debian buster with openhabian (openHAB 2.5.3-1)

--- a/bundles/org.openhab.binding.boschindego/pom.xml
+++ b/bundles/org.openhab.binding.boschindego/pom.xml
@@ -12,11 +12,33 @@
 
   <name>openHAB Add-ons :: Bundles :: Bosch Indego Binding</name>
 
+  <properties>
+    <dep.noembedding>httpclient-osgi,httpcore-osgi,commons-codec</dep.noembedding>
+  </properties>
+
   <dependencies>
     <dependency>
       <groupId>de.zazaz.iot.bosch.indego</groupId>
       <artifactId>bosch-indego-controller-lib</artifactId>
       <version>0.8</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient-osgi</artifactId>
+      <version>4.5.5</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpcore-osgi</artifactId>
+      <version>4.4.9</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
+      <version>1.10</version>
       <scope>compile</scope>
     </dependency>
   </dependencies>

